sdk/storage/arm-storage/README.md

```diff
@@ -1,90 +1,108 @@
 ## Azure StorageManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for StorageManagementClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for StorageManagementClient.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
```

Comment:
We are releasing an update to the code generator today that will result in the below changes. Suggesting this here so that you don't have to re-generate

```suggestion
- [LTS versions of Node.js](https://nodejs.org/about/releases/)
- Latest versions of Safari, Chrome, Edge, and Firefox.
```